<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Devine</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="devine.qrc">:/images/filter.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="the_filter"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTreeWidget" name="the_tree">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QTreeView::item { padding-right: 10px;}.</string>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string notr="true"/>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="the_properties"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionDevices_by_type"/>
    <addaction name="actionDevices_by_connection"/>
    <addaction name="separator"/>
    <addaction name="actionShow_hidden_devices"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_columns"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_Action">
    <property name="title">
     <string>&amp;Action</string>
    </property>
    <addaction name="the_uninstall"/>
    <addaction name="the_enable"/>
    <addaction name="the_disable"/>
    <addaction name="separator"/>
    <addaction name="action_Scan_for_hardware_changes"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Action"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionDevices_by_type">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>D&amp;evices by type</string>
   </property>
  </action>
  <action name="actionDevices_by_connection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>De&amp;vices by connection</string>
   </property>
  </action>
  <action name="actionShow_hidden_devices">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sho&amp;w hidden devices</string>
   </property>
  </action>
  <action name="actionSelect_columns">
   <property name="text">
    <string>Select &amp;columns</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Scan_for_hardware_changes">
   <property name="text">
    <string>&amp;Scan for hardware changes</string>
   </property>
  </action>
  <action name="the_uninstall">
   <property name="text">
    <string>Uninstall device(s)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="the_enable">
   <property name="text">
    <string>Enable device(s)</string>
   </property>
  </action>
  <action name="the_disable">
   <property name="text">
    <string>Disable device(s)</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="the_properties">
   <property name="text">
    <string>&amp;Properties</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>&amp;Export</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>&amp;Import</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="devine.qrc"/>
 </resources>
 <connections/>
</ui>
